### DISCUSSION TITLE
一键部署“Runtime error”问题

### DISCUSSION THREAD

**Discussion by PandaYH** [GaiZhenbiao/ChuanhuChatGPT#406] [Q&A]
复制项目到自己的空间，构建的时候一直失败，没有其他报错提示，如下图所示：
![image](https://user-images.githubusercontent.com/52625464/227785888-9be756c9-b159-453f-ad3a-30b1a5fb351d.png)
![image](https://user-images.githubusercontent.com/52625464/227786051-44c03617-73e1-465a-b432-9ea7ce3c5208.png)

**Comment by Keldos-Li** [comment]
https://b23.tv/H8lbDQG

“现在正常啦”

**Comment by PandaYH** [comment]
额 又是不一样的情况了：
![image](https://user-images.githubusercontent.com/52625464/227824180-cf810953-12f5-4663-b6ae-52c433af04f7.png)

![image](https://user-images.githubusercontent.com/52625464/227824074-73b6d0e9-3753-4c4a-b56b-b6fc141c3bb1.png)

**Reply by GaiZhenbiao** [reply]
忽略这个警告，复制就行了。没问题的。

**Reply by PandaYH** [reply]
感谢！解决了，有个昨天就发现的问题，是加了代码验证码，做了一点改动，然后api输入框就消失了，请问是什么原因：
![image](https://user-images.githubusercontent.com/52625464/227829064-4b6dc3eb-823d-490f-8282-650e8e32edaf.png)

**Reply by GaiZhenbiao** [reply]
啊？什么意思？

**Comment by pageLeo** [comment]
现在又不正常啦，我还以为是我改了部分代码中文字的原因，今天直接复制后跑，还是Runtime error，和最上第一张图一样

**Reply by GaiZhenbiao** [reply]
这个应该是HG的问题。多factory reboot几次试试

**Reply by pageLeo** [reply]
好的，又试了几次，一会能用一会不能hhh
